lib/Doctrine/Common/Cache/RedisCache.php

```diff
@@ -175,10 +175,6 @@ protected function doGetStats()
      */
     protected function getSerializerValue()
     {
-        if (defined('HHVM_VERSION')) {
```

Comment:
This is the exact same change as in https://github.com/doctrine/cache/pull/217 and it's useless since `extension_loaded('igbinary')` is checked